docker-entrypoint.sh

```diff
@@ -61,6 +61,11 @@ pipelinePath=/data/xnat/pipeline
 prearchivePath=/data/xnat/prearchive
 initialized=true
 EOF
+
+  if [ -n "${LDAP_HOST}" ]; then
+    echo "enabledProviders=ldap" >> /data/xnat/home/config/prefs-init.ini
+    echo "enabledProviders=localdb" >> /data/xnat/home/config/prefs-init.ini
```

Comment:
Turns out if you duplicate the key it'll turn the value into a list with each of the options. I _think_ the `prefs-init.ini` file is only read on the initial boot. If the database has already been initialized it will not be read.

`xnat-web` does mention a `prefs-override.ini` file, but I think it's an alternate config file name.